user-interface/src/case-detail/panels/CaseDetailAuditHistory.test.tsx

```diff
@@ -1,10 +1,13 @@
-import { render, screen } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
 import CaseDetailAuditHistory from '@/case-detail/panels/CaseDetailAuditHistory';
-import { MockData } from '@common/cams/test-utilities/mock-data';
-import { CaseHistory } from '@common/cams/history';
 import { CaseAssignment } from '@common/cams/assignments';
+import { CaseHistory } from '@common/cams/history';
 import { ConsolidationOrder } from '@common/cams/orders';
+import MockData from '@common/cams/test-utilities/mock-data';
+import Chapter15MockApi from '@/lib/models/chapter15-mock.api.cases';
 
+// TODO: Case History is now loaded in from a useEffect inside CaseDetailAuditHistory
```

Comment:
Consider.

Reply:
This is done.  Probably shouldn't be a "TODO".  Not sure what we should consider.

Reply:
Yeah.  I think this is a stale comment.  Need to throw it out like 2 year old matzah.